keycloak-apikey/src/main/java/eu/europeana/keycloak/validation/service/SesstionTrackerUpdator.java

```diff
@@ -0,0 +1,92 @@
+package eu.europeana.keycloak.validation.service;
+
+import eu.europeana.keycloak.KeycloakUtils;
+import eu.europeana.keycloak.validation.datamodel.ErrorMessage;
+import eu.europeana.keycloak.validation.datamodel.SessionTracker;
+import eu.europeana.keycloak.validation.util.Constants;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.BiFunction;
+
+/**
+ *
+ */
+public class SesstionTrackerUpdator implements BiFunction<String, SessionTracker, SessionTracker>,
+    Serializable {
+  AtomicReference<ErrorMessage> resultReference = new AtomicReference<>();
```

Comment:
Please move this a bit lower to the instantiation of the other class variables (see also Sonar warnings)